README.md

```diff
@@ -1,108 +1,63 @@
-Please raise any questions or queries there. Contributions welcome!
+When running the application for the first time, run the following command:
 
-Our security policy is located [here](https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/security/policy). 
+```bash
+script/setup
+```
 
-More information about the template project including features can be found [here](https://dsdmoj.atlassian.net/wiki/spaces/NDSS/pages/3488677932/Typescript+template+project).
+This will tear down and setup the application, create .env files and bootstrap the application.
 
-## Creating a CloudPlatform namespace
+If you're coming back to the application after a certain amount of time, you can run:
 
-When deploying to a new namespace, you may wish to use this template typescript project namespace as the basis for your new namespace:
+```bash
+script/bootstrap
+```
 
-<https://github.com/ministryofjustice/cloud-platform-environments/tree/main/namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-accredited-programmes-ui>
+## Running the application
 
-This template namespace includes an AWS elasticache setup - which is required by this template project.
+To start Docker, run all backing services, and then the application itself, run:
 
-Copy this folder, update all the existing namespace references, and submit a PR to the CloudPlatform team. Further instructions from the CloudPlatform team can be found here: <https://user-guide.cloud-platform.service.justice.gov.uk/#cloud-platform-user-guide>
+```bash
+  script/server
+```
 
-## Renaming from HMPPS Template Typescript - github Actions
+## Running the tests
 
-Once the new repository is deployed. Navigate to the repository in github, and select the `Actions` tab.
-Click the link to `Enable Actions on this repository`.
+To run linting, typechecking and the test suite, run:
 
-Find the Action workflow named: `rename-project-create-pr` and click `Run workflow`.  This workflow will
-execute the `rename-project.bash` and create Pull Request for you to review.  Review the PR and merge.
+```bash
+  script/test
+```
 
-Note: ideally this workflow would run automatically however due to a recent change github Actions are not
-enabled by default on newly created repos. There is no way to enable Actions other then to click the button in the UI.
-If this situation changes we will update this project so that the workflow is triggered during the bootstrap project.
-Further reading: <https://github.community/t/workflow-isnt-enabled-in-repos-generated-from-template/136421>
+### Running unit tests
 
-## Manually branding from template app
-Run the `rename-project.bash` and create a PR.
+You can run unit tests individually with:
 
-The rename-project.bash script takes a single argument - the name of the project and calculates from it the project description
-It then performs a search and replace and directory renames so the project is ready to be used.
+```bash
+  npm run test
+```
```

Comment:
And the name of the file? Or do you mean unit tests in isolation of integration tests, rather than individual unit tests?

Reply:
I meant the whole suite of unit tests, but this maybe wasn't clear. Have amended.